### ISSUE TITLE
Conditional container link

### ISSUE THREAD

**Issue by Rokko11** [fabric8io/docker-maven-plugin#1074]
Hi!
Is there a way to ignore the exception, if a linked container does not exists?
I see, there is a flag `leaveUnresolvedIfNotFound`, but I cannot configure this without configuring a custom network.

Background: Sometimes I would like to link a database-container, sometimes I would like to use an existing Database running on another server (and not in docker).

Regards, Rokko

**Comment by rhuss** [comment]
No, there is no support yet for skipping links. 'happy to discuss and accepts PR for this feature though.
